src/chainstate/stacks/index/storage.rs

```diff
@@ -685,28 +1258,38 @@ pub struct TrieStorageConnection<'a, T: MarfTrieId> {
 ///   to the underlying SQL storage. Used internally to simplify
 ///   the TrieStorageConnection/TrieFileStorage interactions
 ///
-struct TrieStorageTransientData<T: MarfTrieId> {
-    last_extended: Option<(T, TrieRAM<T>)>,
+pub struct TrieStorageTransientData<T: MarfTrieId> {
+    /// This is all the nodes written but not yet committed to disk.
```

Comment:
This is the latest trie, right?

Reply:
Yes; it's just renamed from `last_extended` to `uncommitted_writes`, since it's using the new struct `UncommittedState<T>`.